lib/molecules/admin-bar/admin-bar.controller.js

```diff
@@ -61,6 +61,10 @@ function lnMAdminBarController($log, $rootScope, $scope, $cookies, $timeout, $el
       vm.lnLogoutUrl = response.data.logout_url;
       vm.lnEditPageUrl = response.data.edit_page_url;
       vm.lnNewItemTypes = response.data.post_types;
+      vm.lnUserNicename = angular.isDefined( response.data.nicename ) ? response.data.nicename : '';
```

Comment:
But if you're running an older version of the api they won't be there. This was to make it backward compatible, not a breaking change.